src/components/Button/Button.scss

```diff
@@ -124,6 +124,11 @@
   border: none;
   vertical-align: top;
   line-height: normal;
+  font-size: 0;
```

Comment:
I don't know of any better solution, from the looks of it on android it fails gracefully and shows the white space